langchain-core/src/messages/index.ts

```diff
@@ -135,11 +154,6 @@ export abstract class BaseMessage
 
   lc_serializable = true;
 
-  get lc_aliases(): Record<string, string> {
```

Comment:
why did you drop this and the one below?

Reply:
Looks like an error on my part. Those were both added on 20 Dec and I may originally have been working off an older version and then just copied the file over and didn't notice.